src/components/views/elements/Field.tsx

```diff
@@ -60,8 +60,6 @@ interface IProps {
     // If specified, contents will appear as a tooltip on the element and
     // validation feedback tooltips will be suppressed.
     tooltipContent?: JSX.Element | string;
-    // If specified the tooltip will be shown regardless of feedback
-    forceTooltipVisible?: boolean;
```

Comment:
This PR deprecates the last user of `forceTooltipVisible`. Given it's a force flag, it feels reasonably okay to remove it and encourage using mode modern components.